stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -65,6 +68,8 @@
   private ChildNodeRemovedEvent mCachedChildNodeRemovedEvent;
   private ChildNodeInsertedEvent mCachedChildNodeInsertedEvent;
 
+  private final Map<String, List<Integer>> mSearchResults;
```

Comment:
Also, when passing back a collection from Stetho back to chrome, the Json serializer expects a `List`, so using an array and then converting back to a list in the end seems redundant.